### ISSUE TITLE
`null` property breaks `get` for whole feature

### ISSUE THREAD

**Issue by KiwiKilian** [mapbox/mapbox-maps-ios#1914]
## Environment

- Xcode version: 14.2
- iOS version: 16.2
- Devices affected: any
- Maps SDK Version: 10.11.0 / main branch

## Observed behavior and steps to reproduce

`get` expressions break when a feature of a GeoJSON Source is `null`. Take this GeoJSON for example:

```json
{
  "type": "FeatureCollection",
  "features": [
    {
      "type": "Feature",
      "properties": {
        "color": "red"
      },
      "geometry": {
        "type": "Point",
        "coordinates": [-77.09, 38.9]
      }
    },
    {
      "type": "Feature",
      "properties": {
        "color": "green",
        "otherValue": null
      },
      "geometry": {
        "type": "Point",
        "coordinates": [-77.08, 38.9]
      }
    }
  ]
}
```
And a layer styled like this:

```swift
        circleLayer.circleColor = .expression(Exp(.get) { "color" })
        circleLayer.circleRadius = .constant(16)
```

Results in the following:
<img src="https://user-images.githubusercontent.com/18724559/220035684-c8427e3d-dc8d-4dfe-bb06-c0c8e36936ca.png"  width="393" height="852">

## Expected behavior
The black circle should be green. This does work, when I remove or change the `otherValue` property to something other then `null`:
<img src="https://user-images.githubusercontent.com/18724559/220035707-3336b2bf-3627-4354-9061-d478693c6e65.png"  width="393" height="852">

## Notes / preliminary analysis

Here is the full reproduction Swift Code, I used `MultipleGeometriesExample.swift`, use the JSON at the top and put it in `GeoJSONSourceExample.json`.

```swift
import UIKit
import MapboxMaps

@objc(MultipleGeometriesExample)
public class MultipleGeometriesExample: UIViewController, ExampleProtocol {
    enum Constants {
        static let geoJSONDataSourceIdentifier = "geoJSON-data-source"
    }
    internal var mapView: MapView!

    override public func viewDidLoad() {
        super.viewDidLoad()

        // Set the center coordinate and zoom level.
        let centerCoordinate = CLLocationCoordinate2D(latitude: 38.9, longitude: -77.085)
        let options = MapInitOptions(cameraOptions: CameraOptions(center: centerCoordinate, zoom: 11))

        mapView = MapView(frame: view.bounds, mapInitOptions: options)
        mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
        view.addSubview(mapView)

        // Allow the view controller to receive information about map events.
        mapView.mapboxMap.onNext(event: .mapLoaded) { _ in
            self.addGeoJSONSource()
            self.addPointLayer()

            // The below line is used for internal testing purposes only.
            self.finish()
        }
    }

    // Load GeoJSON file from local bundle and decode into a `FeatureCollection`.
    internal func decodeGeoJSON(from fileName: String) throws -> FeatureCollection? {
        guard let path = Bundle.main.path(forResource: fileName, ofType: "geojson") else {
            preconditionFailure("File '\(fileName)' not found.")
        }

        let filePath = URL(fileURLWithPath: path)

        var featureCollection: FeatureCollection?

        do {
            let data = try Data(contentsOf: filePath)
            featureCollection = try JSONDecoder().decode(FeatureCollection.self, from: data)
        } catch {
            print("Error parsing data: \(error)")
        }

        return featureCollection
    }

    private func addGeoJSONSource() {
        // Attempt to decode GeoJSON from file bundled with application.
        guard let featureCollection = try? decodeGeoJSON(from: "GeoJSONSourceExample") else { return }

        // Create a GeoJSON data source.
        var geoJSONSource = GeoJSONSource()
        geoJSONSource.data = .featureCollection(featureCollection)
        try! mapView.mapboxMap.style.addSource(geoJSONSource, id: Constants.geoJSONDataSourceIdentifier)
    }

    public func addPointLayer() {
        // Create a circle layer associated with the GeoJSON data source,
        // filter it so that only the point data is shown,
        // and apply basic styling to it.
        var circleLayer = CircleLayer(id: "circle-layer")

        circleLayer.source = Constants.geoJSONDataSourceIdentifier
        circleLayer.circleColor = .expression(Exp(.get) { "color" })
        circleLayer.circleRadius = .constant(16)
        try! mapView.mapboxMap.style.addLayer(circleLayer)
    }
}
```

**Comment by persidskiy** [comment]
Hi @KiwiKilian, thank you for the report! I created an internal ticket to fix it.

**Comment by somasekharkakarla** [comment]
Hi @persidskiy , any update on this issue?. We are facing too many tickets from iPhone users on our app.

**Comment by somasekharkakarla** [comment]
Hello Guys, any update on this issue. It's been nearly a month?

**Comment by Rehan-sudo** [comment]
Guys, 

It's been too long and the issue still exists. Can you please look into it and resolve as early as possible?

**Comment by lucadegasperi** [comment]
Experiencing the same issue here. This is a pretty big deal imho.

**Comment by OdNairy** [comment]
Thanks for your head-ups. The issues was detected and fixed. It will be included in the upcoming 10.13.0-beta.
